### DISCUSSION TITLE
filtron service

### DISCUSSION THREAD

**Discussion by Kc9jjj** [searx/searx#3281] [Installation issue: Official wiki]
![Screenshot 2022-07-08 11 48 05](https://user-images.githubusercontent.com/89176513/178029361-5b4052e8-eafd-406e-9da5-a9741f0c69f5.png)
I keep getting this error in ubuntu22.04 on wsl2 i've tried everything I can think of I need a little help thanks
![Screenshot 2022-07-08 11 48 05](https://user-images.githubusercontent.com/89176513/178029928-c58fa6d4-67fe-4a1b-83ef-17a85542518a.png)

**Comment by unixfox** [comment]
it's simple searx-docker doesn't support WSL. Install searx with another method.

**Comment by Kc9jjj** [comment]
I’m not using docker I’m following the wiki instructions

73,
Darren Tomblin KC9JJJ

On Jul 8, 2022, at 12:17 PM, Émilien Devos ***@***.***> wrote:

﻿

it's simple searx-docker doesn't support WSL. Install searx with another method.

**Reply by unixfox** [reply]
Ok it's the same case, it's not supported on WSL because WSL is not a real Linux.

You should install a real Linux VM.

**Comment by Kc9jjj** [comment]
What’s the difference it looks like a real Linux to me

73,
Darren Tomblin KC9JJJ

On Jul 8, 2022, at 5:04 PM, Émilien Devos ***@***.***> wrote:

﻿

Ok it's the same case, it's not supported on WSL because WSL is not a real Linux.

You should install a real Linux VM.
